LeanplumTests/LPUserApiTests.m

```diff
@@ -45,4 +54,43 @@ - (void)testUserApi {
     }];
 }
 
+- (void)testUserApiWithHTTPError {
+    XCTestExpectation *expectation = [self expectationWithDescription:@"Query timed out."];
+    LPAPIConfig *lpApiConfig = [LPAPIConfig sharedConfig];
+    [lpApiConfig setAppId:APPLICATION_ID withAccessKey:DEVELOPMENT_KEY];
```

Comment:
Can you please help in streamlining the error class. If you can move the common code to "setup" for lpApiConfig etc.? That will help.
 "LPAPIConfig *lpApiConfig = [LPAPIConfig sharedConfig];
    [lpApiConfig setAppId:APPLICATION_ID withAccessKey:DEVELOPMENT_KEY]; and if anything else"